vello_encoding/src/encoding.rs

```diff
@@ -240,6 +240,12 @@ impl Encoding {
         encoder.finish(true) != 0
     }
 
+    pub fn encode_empty_shape(&mut self) {
```

Comment:
This needs a doc comment.

Reply:
This is still the case

Reply:
Duh, I had forgot to Cmd-S my buffer. Sorry about that; it feels like today is a day I shouldn't be operating heavy machinery.